src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -116,15 +123,21 @@ public static void loadPlugins(final Map<String, String> params) {
                     }
                     if ("theme".equals(m.getMainAttributes().getValue("Plugin-Category"))) {
                         String target = mu.toString();
-                        for (URL url : urls) {
+                        for (URL url : urlList) {
                             if (target.contains(url.toString())) {
                                 THEME_PLUGIN_JARS.add(url);
                             }
                         }
                     }
                     loadFromManifest(m, pluginsClassLoader);
+                } catch (ClassNotFoundException e) {
```

Comment:
A protection here is not perfect.
When target plugin is for theme and `loadFromManifest` raises exception,  `THEME_PLUGIN_JARS` contains the broken plugin. `if ("theme"....` block should be after `loadFromManifest()` to avoid the case.